packages/components/pie-thumbnail/test/component/pie-thumbnail.spec.ts

```diff
@@ -17,4 +33,74 @@ test.describe('PieThumbnail - Component tests', () => {
         // Assert
         expect(thumbnail).toBeVisible();
     });
+
+    backgroundColorPropOptions.forEach(({ backgroundColor }) => {
+        test(`background color should be successfully set to "${backgroundColor}"`, async ({ page }) => {
+            const props: Partial<ThumbnailProps> = {
+                backgroundColor,
+            };
+            const pieThumbnailPage = new BasePage(page, 'thumbnail--default');
+            await pieThumbnailPage.load(props);
+
+            const thumbnailComponent = page.locator(componentSelector);
+
+            const [currentBgStyle, expectedBgStyle] = await getShadowElementStylePropValues(thumbnailComponent, componentSelector, ['background-color', backgroundColor as string]);
+
+            expect(currentBgStyle).toBe(expectedBgStyle);
```

Comment:
Rather than using `getShadowElementStylePropValues` (which we've been trying to reduce usage of and eventually remove once our test refactor is complete), you may be able to leverage:

`await expect(thumbnailComponent).toHaveCSS('background-color, expectedBgStyle);`